rb/lib/selenium/webdriver/common/window.rb

```diff
@@ -156,6 +156,27 @@ def minimize
       def full_screen
         @bridge.full_screen_window
       end
+
+      #
+      # Get current screen resolution
```

Comment:
This technically is wrong as it is returning individual dimensions, not the resolution. i.e. this returns 1024, not 1024x768.

Is there a reason you have these specified into individual items?

Why not return the resolution as either a hash / array with the two items below set to private?

Also is there a reason you've not queried the `#size` method - And gone down the JS route?